src/appsettings.template.json

```diff
@@ -0,0 +1,40 @@
+{
+    "AppSettings": {
+        "BeginAllowedSchoolYear": 2022,
+        "EndAllowedSchoolYear": 2035,
+        "AuthenticationService": "http://dev-test-auth-service",
```

Comment:
We need to use our fake token endpoint here. That means we must be able to inject an environment variable with the host address, and then append "/oauth/token".

We can make this change in the bulk load spike instead of in this ticket, since that's when we will really need it.